### ISSUE TITLE
Assertion Failed: You must define a `min` for MinValidator

### ISSUE THREAD

**Issue by pjcarly** [vladaspasic/ember-cli-data-validation#2]
I'm getting this error trying to define the min property on a model attribute:

``` javascript
var Account = DS.Model.extend(ValidatorMixin, {
  name: DS.attr('string', {
    validation: {
      required: true,
      min: 5
    }
  })
});
```

**Comment by vladaspasic** [comment]
Hi @pjcarly ,

this should be now fixed with this commit ad96284d7a37408daa9bf3441f01f77f2b62565f 

You should now upgrade the addon to version `0.0.10`.

**Comment by pjcarly** [comment]
It is working now, thanks for the quick fix!
